### PR TITLE
upgraded to RT 7.27.10 and xray 3.35.0

### PR DESCRIPTION
**Pull request by giri-vsr**
AWS | Update EC2 Quickstart RT and xray to 7.27.10 and 3.35.0
https://jira.jfrog.org/browse/PTRENG-3253

### CODE REVIEW

File: .jfrog.taskcat.yml [hunk 1]
```diff
@@ -3,32 +3,32 @@ project:
   owner: quickstart-eng@amazon.com
   s3_object_acl: private
   s3_regional_buckets: True
-  s3_bucket: tcat-422383ecc658557f9a377abae675aac0  # commercial accounts (also for GovCloud)
+  s3_bucket: tcat-34c17b83dcbd547393a86d8053a8d298 # commercial accounts (also for GovCloud)
```
Comment (alexhung) [inline .jfrog.taskcat.yml:6]: Not specific to this PR. We need to move away from using Taskcat created bucket for testing. This is an known issue that CSO office has asked us to resolve for a while now. We should create a permanent bucket for testing purpose.

File: .jfrog.taskcat.yml [hunk 2]
```diff
@@ -3,32 +3,32 @@ project:
...
     ProvisionBastionHost: "Enabled"
-    NumberOfSecondary   : 2
-    InstallXray         : "true"
+    NumberOfSecondary: 2
+    InstallXray: "true"
     XrayNumberOfSecondary: 1
 
     # RemoteAccessCidr  : "0.0.0.0/0"
-    RemoteAccessCidr  : "107.3.154.154/32"  # vinay's home IP (with or without zscalar)
+    RemoteAccessCidr: "49.207.208.42/32" # vinay's home IP (with or without zscalar)
```
Comment (alexhung) [inline .jfrog.taskcat.yml:31]: See comment on override file

**Review by alexhung — CHANGES_REQUESTED** [review]

**Comment by alexhung** [comment]
Please include JIRA ticket ID in the PR title, better yet include the link to the JIRA ticket in the description.

File: README.md [hunk 3]
```diff
@@ -2,6 +2,105 @@
 
 For architectural details, step-by-step instructions, and customization options, see the [deployment guide](https://aws-quickstart.github.io/quickstart-jfrog-artifactory/).
 
-To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo. 
+To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
 
 To submit code for this Quick Start, see the [AWS Quick Start Contributor's Kit](https://aws-quickstart.github.io/).
+
+## Upgrading RT and Xray Version
```
Comment (alexhung) [inline README.md:9]: Not sure if we want to include these here in a public repo. Sorry if I wasn't clear.

File: .jfrog.taskcat.yml [hunk 4]
```diff
@@ -3,14 +3,14 @@ project:
   owner: quickstart-eng@amazon.com
   s3_object_acl: private
   s3_regional_buckets: True
-  s3_bucket: tcat-34c17b83dcbd547393a86d8053a8d298 # commercial accounts (also for GovCloud)
-  #s3_bucket: tcat-a3e80b6745b2547da1c745b16adf2a66    # aws-seller accounts (MarketPlace)
+  s3_bucket: tcat-qs-ec2 # commercial accounts (also for GovCloud)
+  #s3_bucket: tcat-qs-ec2    #aws-seller accounts (MarketPlace)
```
Comment (alexhung) [inline .jfrog.taskcat.yml:7]: Not sure if we can/should share same S3 bucket between account. Should be fine as long as the sub-directories are unique for each account.

File: .jfrog.taskcat.yml [hunk 5]
```diff
@@ -3,14 +3,14 @@ project:
...
 
   parameters:
-    KeyPairName: "giri"
+    #KeyPairName: "" #Key Pair Name which should be already exists in aws
```
Comment (alexhung) [inline .jfrog.taskcat.yml:10]: Should mention the use of Taskcat override file here in the comment.

**Review by alexhung — CHANGES_REQUESTED** [review]

File: .jfrog.taskcat.yml (hunk 4 above)
Comment (giri-vsr) [reply .jfrog.taskcat.yml:7]: yeah I thought of using same s3 bucket for all but sometimes when testing if it fails we make changes and run another while the existing stack is getting removed.Also taskcat uploads with the same name by default

Comment (alexhung) [reply .jfrog.taskcat.yml:7]: Ok fine for now.

File: README.md (hunk 3 above)
Comment (giri-vsr) [reply README.md:9]: Revert it?

File: .jfrog.taskcat.yml (hunk 5 above)
Comment (giri-vsr) [reply .jfrog.taskcat.yml:10]: Sure

File: README.md (hunk 3 above)
Comment (alexhung) [reply README.md:9]: Yes, I think that's best.

Comment (giri-vsr) [reply README.md]: Done.

File: .jfrog.taskcat.yml (hunk 2 above)
Comment (giri-vsr) [reply .jfrog.taskcat.yml]: Done.

**Review by alexhung — APPROVED** [review]

**Pull request merged**